index.html

```diff
@@ -1289,36 +1299,94 @@ <h2>Dereferencing the Resource</h2>
+									property or the <code>service</code> DID parameter or both contain relative
+									references, the corresponding absolute URIs MUST be resolved and used for determining
+									the match, using the rules specified in <a data-cite="RFC3986#section-5">RFC3986 Section 5: Reference Resolution</a>
+									and in section <a data-cite="did-core#relative-did-urls">Relative DID URLs</a> in [[[DID-CORE]]].</li>
+								<li>If the <var>input <a>DID URL</a></var> contains the
+									<a data-cite="did-core#did-parameters">DID parameter</a> <code>serviceType</code>:
+									Select the <a data-cite="did-core#services">service</a> if its <code>type</code>
+									property matches the value of the <code>serviceType</code> DID parameter.</li>
+							</ol>
+							The selected <a data-cite="did-core#services">services</a> are a list called the <var>selected <a>services</a></var>.
+						</li>
+						<li>If the <var>input <a>DID URL</a></var> contains the
+							<a data-cite="did-core#did-parameters">DID parameter</a> <code>relativeRef</code>:
+							<ol class="algorithm">
+								<li>For each <var>selected <a>service</a></var>:
+									<ol class="algorithm">
+										<li>If the value of the <code>serviceEndpoint</code> property of the <var>selected <a>service</a></var>
+										is a <a data-cite="INFRA#maps">map</a>, skip this <var>selected <a>service</a></var>.</li>
+										<li>If the value of the <code>serviceEndpoint</code> property of the <var>selected <a>service</a></var>
+											is a <a data-cite="INFRA#string">string</a>, add this value to a list of
+											<var>selected <a>service endpoint</a> URLs</var>.</li>
+										<li>If the value of the <code>serviceEndpoint</code> property of the <var>selected <a>service</a></var>
+											is a <a data-cite="INFRA#ordered-set">set</a>, add all its items that are <a data-cite="INFRA#string">strings</a>
+											to a list of <var>selected <a>service endpoint</a> URLs</var>.</li>
+									</ol>
+								</li>
+								<li>For each <var>selected <a>service endpoint</a> URL</var>, execute the algorithm specified in
+									<a data-cite="RFC3986#section-5">RFC3986 Section 5: Reference Resolution</a> as follows:
+									<ol class="algorithm">
+										<li>The <strong>base URI</strong> value is the <var>selected <a>service endpoint</a> URL</var>.</li>
+										<li>The <strong>relative reference</strong> is the value of the
+											<a data-cite="did-core#did-parameters">DID parameter</a> <code>relativeRef</code>.</li>
+										<li>Update the <var>selected <a>service endpoint</a> URL</var> to
+										the result of the "Reference Resolution" algorithm.</li>
+									</ol>
+								</li>
+							</ol>
+						</li>
+						<li>If the <b>accept</b> <var>input <a href="#did-url-dereferencing-options">DID dereferencing option</a></var>
+							is missing, or if its value is the Media Type of a <a>representation</a> of a <a>DID document</a>:
```

Comment:
Wondering about the Media Type here. I thought we said there was only one media type for a DID document - `application/did`

Reply:
One that we define in our specs. But my understanding is that anyone can also define different DID document representations with different media types, e.g. CBOR based.